docs/deployment/application-deployment.md

```diff
@@ -109,23 +109,23 @@ If you only want to rebuild and tag a container, you can skip the deployment pha
 
 ``` shell
 # on the Dokku host
-dokku config:set ruby-rails-sample DOKKU_SKIP_DEPLOY=true
+dokku config:set rails-sample-app DOKKU_SKIP_DEPLOY=true
 ```
 
 ### Redeploying or restarting
 
-If you need to redeploy (or restart) your app: 
+If you need to redeploy or restart your app: 
 
 ```shell
 # on the Dokku host
-dokku ps:rebuild ruby-rails-sample
+dokku ps:rebuild rails-sample-app
 ```
 
 See the [process scaling documentation](/docs/deployment/process-management.md) for more information.
 
-### Deploying with private git submodules
+### Deploying with private `git` submodules
 
-Dokku uses Git locally (i.e. not a Docker image) to build its own copy of your app repo, including submodules. This is done as the `dokku` user. Therefore, in order to deploy private Git submodules, you'll need to drop your deploy key in `/home/dokku/.ssh/` and potentially add `github.com` (or your VCS host key) into `/home/dokku/.ssh/known_hosts`. The following test should help confirm you've done it correctly.
+Dokku uses `git` locally (i.e. not a Docker image) to build its own copy of your app repo, including submodules, as the `dokku` user. This means that, in order to deploy private `git` submodules, you need to put your deploy key in `/home/dokku/.ssh/` and potentially add `github.com` (or your VCS host key) into `/home/dokku/.ssh/known_hosts`. You can use the following test to confirm your setup is correct:
```

Comment:
We're referring to Git the project, not `git` the tool, in this paragraph, so please revert to `Git` instead of the code version :)